### PR TITLE
Add twitter card

### PR DESCRIPTION
**Pull request by mmahalwy**

### CODE REVIEW

File: src/scripts/components/Html.js [hunk 1]
```diff
@@ -20,6 +20,11 @@ class Html extends React.Component {
           <meta property="og:url" content="http://quran.com/" />
           <meta property="og:image" content="http://quran.com/images/thumbnail.png" />
 
+          <meta name="twitter:card" content="summary_large_image" />
+          <meta name="twitter:site" content="@nytimes" />
```
Comment (ahmedre) [inline src/scripts/components/Html.js]: need to replace this with an account - if we don't have one still, use QuranAndroid for now maybe?

**Comment by ahmedre** [comment]
looks good to me - it would be really really really cool if we could dynamically generate `twitter:image`, such that if we are only showing one ayah, just use the url for the retina image for that ayah.

**Comment by muhsinali** [comment]
Do we have an official twitter account? If not I can make it just let me know ikhwa.

File: src/scripts/components/Html.js (hunk 1 above)
Comment (mmahalwy) [inline src/scripts/components/Html.js]: Yeah sounds good to me! I will file the Twitter thing today

**Comment by mmahalwy** [comment]
@muhsinali all the ones we should have are taken... so i will file with twitter a claim. Especially @quran_com

**Comment by mmahalwy** [comment]
@ahmedre I agree, it'd be cool to dynamically generate them. Where are the retina images?

**Comment by ahmedre** [comment]
http://quran.com/images/ayat_retina/{sura}_{ayah}.png

ex:
http://quran.com/images/ayat_retina/1_1.png
http://quran.com/images/ayat_retina/2_14.png

if these are too wide, can probably generate less wide ones.

**Comment by mmahalwy** [comment]
@ahmedre i think social media auto adjusts it.

**Comment by mmahalwy** [comment]
closing for now

**Pull request closed without merging**